gcloud/monitoring/client.py

```diff
@@ -164,8 +172,7 @@ def metric_descriptor(self, type_,
         :param value_type:
             The value type of the metric. It must be one of
             :data:`ValueType.BOOL`, :data:`ValueType.INT64`,
-            :data:`ValueType.DOUBLE`, :data:`ValueType.STRING`,
-            or :data:`ValueType.DISTRIBUTION`.
```

Comment:
I don't think we should drop `DISTRIBUTION` from this list of allowed values for metric descriptors, even if we don't advertise it in the docstrings for the factory methods used to send custom metric data.